xbmc/threads/Event.cpp

```diff
@@ -115,11 +105,9 @@ namespace XbmcThreads
     // signaled and sets 'signaled' to the first one it
     // finds.
     // ==================================================
-    signaled = NULL;
-    for (std::vector<CEvent*>::iterator iter = events.begin();
-         signaled == NULL && iter != events.end(); ++iter)
+    signaled = nullptr;
+    for (auto* cur : events)
```

Comment:
Should not be required to specify * here, auto will deduce it since the vector members are pointers. Doesn't hurt, just a tip.